src/Platforms/AbstractPlatform.php

```diff
@@ -731,18 +731,21 @@ public function getNowExpression(): string
      *
      * Note: Not SQL92, but common functionality.
      *
-     * @param string      $string SQL expression producing the string from which a substring should be extracted.
-     * @param string      $start  SQL expression producing the position to start at,
-     * @param string|null $length SQL expression producing the length of the substring portion to be returned.
-     *                            By default, the entire substring is returned.
+     * SQLite only supports the 2 parameter variant of this function.
+     *
+     * @param string   $string An sql string literal or column name/alias.
+     * @param int      $start  Where to start the substring portion.
+     * @param int|null $length The substring portion length.
+     *
+     * @return string
      */
-    public function getSubstringExpression(string $string, string $start, ?string $length = null): string
+    public function getSubstringExpression($string, $start, $length = null)
```

Comment:
Ah I think this is where all the issues come from, isn't it? :sweat_smile: bad merge resolution.

Reply:
We don't, that's why I say "bad merge resolution", and why this diff is marked as outdated :smile: sorry for the confusion.

Reply:
Sorry, I didn't notice the Outdated pill.

Reply:
No worries :)